src/Tasks/Microsoft.NET.Build.Extensions.Tasks/GetDependsOnNETStandard.net46.cs

```diff
@@ -42,8 +43,9 @@ internal static bool GetFileDependsOnNETStandard(string filePath)
             {
                 // on Windows use CLR's unmanaged metadata API.
                 // Create the metadata dispenser and open scope on the source file.
+                var filePathAbsolute = Path.GetFullPath(filePath);
```

Comment:
@dsplaisted do you think we should do this here or in https://github.com/dotnet/sdk/blob/master/src/Tasks/Common/src/ItemUtilities.cs#L113?

Also, it may be better to first check `IsPathRooted` first for performance.  The MSBuild code this is based off of does that: https://github.com/Microsoft/msbuild/blob/ab090d1255caa87e742cbdbc6d7fe904ecebd975/src/Tasks/AssemblyDependency/ReferenceTable.cs#L407-L414